### PR TITLE
fix #80288 tvmovie.de

### LINKED ISSUE #80288: www.tvmovie.de
### Issue URL (Ads)
 [https://www.tvmovie.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.tvmovie.de%2F)
### Comment
Username: @Morku
> Ad placeholder

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#80288

### CODE REVIEW

File: GermanFilter/sections/specific.txt [hunk 1]
```diff
@@ -79,6 +79,7 @@ wiesbadenaktuell.de,rauchmeldungen.de,simfans.de,wn.de,cosmopolitan.de,wallstree
 ! ##.id-Container--proBEEP
 hallo-muenchen.de,az-online.de,lokalo24.de,rosenheim24.de,kreiszeitung.de,nordbuzz.de,mannheim24.de,merkur.de,oktoberfest-live.de,tz.de##.id-Container--proBEEP
 !
+tvmovie.de###dikr-responsive-ads-pb
```
Comment (BlazDT) [inline GermanFilter/sections/specific.txt:82]: Element has style `display: block !important;` and overwrites usually our rule (modifying style to position -3000 should work though).
Though I do not see that left-over at all.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: GermanFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply GermanFilter/sections/specific.txt:82]: @BlazDT Use AdGuard Apps.

Comment (wout-codes) [reply GermanFilter/sections/specific.txt:82]: This rule is enough on my end.

Comment (BlazDT) [reply GermanFilter/sections/specific.txt:82]: Hmm, after reopening site in incognito mode the !important part of style is gone and rule works. Strange.

**Review by BlazDT — APPROVED** [review]

**Pull request merged**